articles/server-apis/aspnet-webapi/authenticate.md

```diff
@@ -46,7 +46,7 @@ ${snippet(meta.snippets.setup)}
 Set the respective values.
 
 ### 4. Securing your API
```

Comment:
Hey @camimonse , while you are at it, can you change the header to `Secure your API`? Thanks!
([styleguide reference](https://github.com/auth0/docs/blob/master/STYLEGUIDE.md#gerunds))